lib/loggerator/loggerator.rb

```diff
@@ -24,12 +24,17 @@ def log(data, &block)
     Log.to_stream(Log.stdout, Log.contexts(data), &block)
   end
 
-  def log_error(e=$ERROR_INFO, data = {})
-    exception_id = e.object_id
+  def log_error(error = $!, data = {})
+    unless error.is_a?(Exception)
```

Comment:
I see your point, and I think you're right too much (∩｀-´)⊃━☆ﾟ.*･｡ﾟ

I'm going to simplify it force the exception.